Update memory-limit.md (#3054)
RedisLabs/redislabs-docs: content/rs/databases/memory-performance/memory-limit.md

Additional factors for  databases with Auto Tiering enabled:

> - The available flash space has to be greater or equal to the total DB size (RAM+Flash). The extra space accounts for write buffers as well as for [write amplification]({{<relref "https://en.wikipedia.org/wiki/Write_amplification">}}).

Reviewer: ```suggestion
- The available flash space must be greater than or equal to the total database size (RAM+Flash). The extra space accounts for write buffers and [write amplification](https://en.wikipedia.org/wiki/Write_amplification).
```
